### PR TITLE
Bump code style to PSR12

### PR DESCRIPTION
**Pull request by GinoPane**

### CODE REVIEW

**Comment by gep** [comment]
@GinoPane Can you please fix the unit tests failing? thnx

File: phpcs.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!-- @see https://pear.php.net/manual/en/package.php.php-codesniffer.annotated-ruleset.php -->
+<ruleset name= "PHPCS Rules">
+  <description>PHPCS ruleset for the DynamoDB ODM</description>
+
+  <file>src</file>
+
+  <!-- Show progress of the run -->
+  <arg value= "p"/>
+
```
Comment (gep) [inline phpcs.xml:10]: Can we also explicitly include PSR12 standard here? Like the following:

```xml
<rule ref="PSR12"/>
```

File: Dockerfile [hunk 2]
```diff
@@ -6,6 +6,10 @@ WORKDIR /application
 
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN apt-get install -y ca-certificates
+COPY ./docker/ssl.crt /usr/local/share/ca-certificates/ssl.crt
+RUN update-ca-certificates
```
Comment (gep) [inline Dockerfile:11]: What do we need those certs for?

Comment (GinoPane) [reply Dockerfile]: Removed. It shouldn't be there. Needed them to overcome proxy issues

File: phpcs.xml (hunk 1 above)
Comment (GinoPane) [reply phpcs.xml:10]: Yep, it should be there

File: src/Model/Serializer/AbstractSerializer.php [hunk 3]
```diff
@@ -157,7 +157,7 @@ function (array $processedValue, ModelInterface $model): array {
                 },
                 []
             ),
-            MoneyType::TYPE_NAME => (int) round(bcmul((string)$value, '100.0', 1)),
+            MoneyType::TYPE_NAME => (int) round((float)bcmul((string)$value, '100.0', 1)),
```
Comment (gep) [inline src/Model/Serializer/AbstractSerializer.php]: Can we trigger deprecation warning for this please? This should not used anymore

**Review by petergodmanap — APPROVED** [review]

**Pull request merged**